### PR TITLE
Updated dependencies to resolve vulnerabilities and removed npm.

### LINKED ISSUE #27: npm defined as a dependency
It seems that in the list of dependencies, npm is also installed as a dependency. I don't think that npm should be a dependency as it should be installed by default on any device that runs node.js. However, since npm is packaged as well, the node_modules directory is excessively large.

This dependency seems to have slipped in at [83bb3c5](https://github.com/Shotbow/ChatBot/commit/83bb3c5c4d685adf2cd9408bf7c327e60e6ea6dd#diff-b9cfc7f2cdf78a7f4b91a753d10865a2).

### PR DESCRIPTION
**Pull request by McJeffr**
There were over 100+ vulnerabilities, mainly caused by nodemon. Also, npm was added as a dependency but was never used. npm should be installed on any platform that also has node.js installed, including it as a dependency will increase the size of the node_modules unnecessarily.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,12 +11,11 @@
   "repository": "https://github.com/Shotbow/ChatBot",
   "dependencies": {
     "config": "^1.30.0",
```
Comment (McJeffr) [inline package.json:13]: I didn't update `config` as it appears that could be a breaking change (1.x.x -> 3.x.x)

**Comment by navarr** [comment]
This increases the Discord.js major version.  Have you tested it?  (And if you're testing we can probably up config as well)

**Comment by McJeffr** [comment]
> This increases the Discord.js major version. Have you tested it? (And if you're testing we can probably up config as well)

I actually missed that. Yeah, let me run some additional checks (sadly we got no unit testing). For config, I think the API is different but I can try it with the latest version.

**Comment by McJeffr** [comment]
I've downgraded the Discord.js version again to the latest version of 11.x.x. This might have introduced warnings about deprecation, these are simply warning us of the road ahead towards updating to 12.x.x, which we will have to do sooner than later. It seems to run without issues. I'll update the issue I have about the change in the Discord API to one about updating to v12 instead.

**Pull request merged**